runtime/ddr/overrides-vm

```diff
@@ -89,6 +89,7 @@ typeoverride.J9EnclosingObject.nameAndSignature=J9SRP(J9ROMNameAndSignature)
 typeoverride.J9ROMClass.className=J9SRP(J9UTF8)
 typeoverride.J9ROMClass.cpShapeDescription=J9SRP(U32)
 typeoverride.J9ROMClass.innerClasses=J9SRP(J9SRP(J9UTF8))
+typeoverride.J9ROMClass.enclosedInnerClasses=J9SRP(J9SRP(J9UTF8))
```

Comment:
Please insert the additions so things remain in alphabetical order (for both `J9ROMClass` and `J9ROMArrayClass`).

Reply:
Updated as suggested.